### PR TITLE
bump version of the SR-IOV operator to 4.4

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

File: cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh [hunk 1]
```diff
@@ -28,14 +28,12 @@ function deploy_sriov_operator {
...
-    # TODO: right now in CI we need to use upstream sriov cni in order to have this
-    # https://github.com/intel/sriov-cni/pull/88 available. This can be removed once the feature will
-    # be merged in openshift sriov operator. We need latest since that feature was not tagged yet
-    sed -i '/SRIOV_CNI_IMAGE/!b;n;c\              value: nfvpe\/sriov-cni' ./deploy/operator.yaml
-    sed -i 's#image: quay.io/openshift/origin-sriov-network-operator$#image: quay.io/openshift/origin-sriov-network-operator:4.2#' ./deploy/operator.yaml
-    sed -i 's#value: quay.io/openshift/origin-sriov-network-config-daemon$#value: quay.io/openshift/origin-sriov-network-config-daemon:4.2#' ./deploy/operator.yaml
+    export RELEASE_VERSION=4.4
+    export SRIOV_NETWORK_OPERATOR_IMAGE=quay.io/openshift/origin-sriov-network-operator:${RELEASE_VERSION}
+    export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=quay.io/openshift/origin-sriov-network-config-daemon:${RELEASE_VERSION}
+    export OPERATOR_EXEC=${KUBECTL}
```
Comment (fedepaol) [inline cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: https://github.com/kubevirt/kubevirtci/pull/134#discussion_r317153351

Comment (phoracek) [reply cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: Now we have multiple environment variables to export and I find this more readable than inline. And I want to keep it consistent. (Not sure I can export SHELL too, can I?)

Comment (fedepaol) [reply cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: > (Not sure I can export SHELL too, can I?)

Yes, please

File: cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh [hunk 2]
```diff
@@ -28,14 +28,12 @@ function deploy_sriov_operator {
   fi
 
   pushd $operator_path
-    # TODO: right now in CI we need to use upstream sriov cni in order to have this
-    # https://github.com/intel/sriov-cni/pull/88 available. This can be removed once the feature will
-    # be merged in openshift sriov operator. We need latest since that feature was not tagged yet
-    sed -i '/SRIOV_CNI_IMAGE/!b;n;c\              value: nfvpe\/sriov-cni' ./deploy/operator.yaml
-    sed -i 's#image: quay.io/openshift/origin-sriov-network-operator$#image: quay.io/openshift/origin-sriov-network-operator:4.2#' ./deploy/operator.yaml
-    sed -i 's#value: quay.io/openshift/origin-sriov-network-config-daemon$#value: quay.io/openshift/origin-sriov-network-config-daemon:4.2#' ./deploy/operator.yaml
+    export RELEASE_VERSION=4.4
```
Comment (dhiller) [inline cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: Indentation looks odd here. Or is it recommended to indent within `pushd` and `popd`?

File: cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh (hunk 1 above)
Comment (phoracek) [reply cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: Done

**Comment by fedepaol** [comment]
I was wondering if it would also make sense to bump up the version of the cluster as it's done in https://github.com/kubevirt/kubevirtci/tree/master/cluster-up/cluster/kind-k8s-1.17.0-ipv6

Maybe in a different pr?

**Comment by phoracek** [comment]
May be worth in this one, just to slim down the time needed for testing with KubeVirt (unless we hit an issue and it would be even more difficult to figure it out)

File: cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh (hunk 2 above)
Comment (phoracek) [reply cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: It is kept from the previous version of the code. I personally prefer to indent to make it clear what is the context of pushd.

**Comment by dhiller** [comment]
/lgtm
/approve

Just in case you need to do sth about it

/hold

**Comment by phoracek** [comment]
I need to test if it works at all.

**Pull request closed without merging**